attributes/default.rb

```diff
@@ -67,7 +51,7 @@
 # This specifies how to connect to a master or a quorum of masters. This flag
 # works with 3 different techniques.  See Mesos documentation for details:
 # http://mesos.apache.org/documentation/latest/configuration
-default['mesos']['slave']['flags']['master']                = '`cat /etc/mesos/zk`'
+default['mesos']['slave']['flags']['master']                = 'localhost:5050'
```

Comment:
Any reason why we should default to doing master discovery via ip:port instead instead of zookeeper?  Personally I don't think we should encourage anyone in pointing their Mesos slaves to anything other than zookeeper for discovering masters.